README.md

```diff
@@ -8,33 +8,37 @@
-
-If you would like to **add to the discussion and submit changes**, please see our [Contributing Guidelines](./CONTRIBUTING.md) and [Code of Conduct](https://github.com/datatogether/datatogether/blob/master/CONDUCT.md). 
+If you would like to **add to the discussion and submit changes**, please see our [Contributing Guidelines](./CONTRIBUTING.md) and [Code of Conduct](https://github.com/datatogether/datatogether/blob/master/CONDUCT.md).
 
 If you want to **get into the code and start hacking**, then [webapp](https://github.com/datatogether/sentry) and [sentry](http://github.com/datatogether/sentry) are the best points of entry.
 
 We use GitHub issues for [tracking ideas and errors](https://github.com/datatogether/datatogether/issues) and Pull Requests (PRs) for [submitting changes](https://github.com/datatogether/datatogether/pulls). Every repository has its own issue tracker so please be sure to post your issues in the best place!
 
-## Mission
+Many of the people behind Data Together are active participants in the [Archivers Slack team](https://archivers-slack.herokuapp.com/), which anyone can join. We use the #datatogether channel.
 
-_Our purpose for existing, our charge, our goal, what this initiative does. (1 succinct sentence only)_
+## Mission (work in progress)
 
-Data Together helps communities steward the data they care about.
+We're building a better future for data.
 
-_[if there’s extra on “how” we do our work, add **[model](/model.md) / strategy / strategic plan / initiative areas**]_
+Data Together is a rallying point for everyone who believes that people should be able to possess copies of the data they rely on, and who see that communities should be using decentralized technologies in order to hold data as collective assets.
 
-## Vision 
+## Vision (work in progress)
 
 _Our future abilities and role in broader movement, NOT a utopian static state. Meant to be inspirational to ourselves and partners to strive together, and can be more flowery than the mission_.
 
-Together we are the civic layer of the decentralized web where communities work cooperatively, share responsibilities, and reinforce each other so that data is accessible to all, immediately discoverable, easily verifiable, and robustly preserved.
+Decentralized, peer to peer technologies present radical opportunities for people and communities to take possession of the data they rely on and to coordinate amongst themselves to govern what data are held (or not held) on their machines and their networks. This will be a powerful tool for addressing many pressing problems of the web today  -- preventing monopolies, addressing issues of trust on the web, preserving the public record, increasing government transparency and encouraging constructive usage of government data, improving reproducibility of scientific research, dealing with censorship, protecting privacy, giving disenfranchised groups control over their own voice on the web, etc.
```

Comment:
can we refer to "Decentralized, peer to peer technologies" as the d web and sneak in a quick definition a little earlier?

Reply:
+1 to defining d web.
my comment about property/juridical language I think applies here with "possession"

Reply:
👍 I will update all the text in this document to reflect that. Once we find language we like, we can apply it everywhere. This overlaps with @ebarry's suggestion in https://github.com/datatogether/datatogether/issues/32